### ISSUE TITLE
NVDA shouldn't announce the landmarks as I tab while in focus mode

### ISSUE THREAD

**Issue by fatihyuksek1** [nvaccess/nvda#12830]
### Is your feature request related to a problem? Please describe.
When tabbing through a webpage while in focus mode, NVDA often tells the names of the landmarks, slowing me down. I use the focus mode because I only need to navigate the links in or buttons on a page, not landmarks etc. I could use the browse mode for that.
### Describe the solution you'd like
When I turn on focus mode, NVDA should stop reporting the landmarks, instead it should report what the focus goes through only, like buttons or links that could be focused with the focus mode. This should speed the navigation while there are a lot of landmarks in a page.
### Describe alternatives you've considered
A checkbox in the browse mode settings dialog could be added instead to make this feature optional.

**Comment by Adriani90** [comment]
I agree reporting regions, baners and landmarks while tabbing in focus mode is not really useful, it just introduces a lot of verbosity.
Reporting landmarks, baners and regions are actually more suitable for quick navigation and while using arrow keys in browse mode.
cc: @jcsteh what was actually the rationale behind introducing reoprting of landmarks and regions when tabbing in focus mode?

**Comment by jcsteh** [comment]
NVDA has always announced semantically relevant containers as focus enters them, whether you're on the web or elsewhere. In desktop apps, groupings and toolbars are good examples of this. This can provide important contextual information. If you have several groups of options with a Learn More link, for example, the name of the group can be important to disambiguate them. It can also help you quickly locate something when tabbing quickly.

Landmarks are also semantically relevant containers, so they're treated the same way.

Personally, I'm not interested in debating this, but as requested, that's the explanation for why this choice was made.
